frontend/integration-tests/tests/event.scenario.ts

```diff
@@ -0,0 +1,56 @@
+import { browser, $, element, by, ExpectedConditions as until } from 'protractor';
+import { appHost, checkLogs, checkErrors, testName } from '../protractor.conf';
+import { safeLoad, safeDump } from 'js-yaml';
+import * as _ from 'lodash';
+import * as crudView from '../views/crud.view';
+import * as yamlView from '../views/yaml.view';
+
+describe('Events', () => {
+  afterEach(() => {
+    checkLogs();
+    checkErrors();
+  });
+
+  describe('Events', () => {
+    const name = `${testName}-pod`;
+    const testLabel = 'automatedTestName';
+
+    it('displays Events all resources', async () => {
+      await browser.get(`${appHost}/k8s/all-namespaces/events`);
+      await browser.wait(until.presenceOf($('.co-sysevent__box')));
+      expect($('.co-sysevent__box').isPresent()).toBe(true);
+    });
+
+    it('creates a new pod resource instance for event testing', async () => {
```

Comment:
It would be simpler to use `kubectl` to create a pod in `beforeAll` and use `kubectl` to delete the pod in `afterAll`. The CRUD tests already cover creating and deleting resources in console really well, so we should keep it simple here. That would avoid duplicating the logic and possible flakes.

Reply:
Oh so much simpler - I was not aware of execSync - the testcases are so much more straight forward now only checking if the events show up for all and for specific namespaces instead of doing crud stuff.